cartoframes/viz/map.py

```diff
@@ -188,8 +191,33 @@ def __init__(self,
     def _repr_html_(self):
         return self._htmlMap.html
 
-    def publish(self):
-        pass
+    def publish(self, name, maps_api_key='default_public', context=None, password=None):
```

Comment:
What's password?

Reply:
You can creare a custom visualization wirh password. It is the same as a embed with password